code/modules/organs/organ_internal.dm

```diff
@@ -108,6 +108,23 @@
 			if (prob(3))	//about once every 30 seconds
 				take_damage(1, prob(30))
 
+	//Process natural and assisted healing
+	var/peridaxon = owner.reagents.has_reagent(/datum/reagent/medicine/peridaxon)
+	var/inaprovaline = owner.reagents.has_reagent(/datum/reagent/medicine/inaprovaline)
+	var/organ_heal_chance = 0
+	var/organ_hurt_chance = 0
```

Comment:
same as above. you dont need.
var/organ_hurt
organ_hurt = number
if(organ_hurt)

instead you can just do

if(number)